data/virt_autotest/guest_unattended_installation_files/sles_12_plus_kvm_hvm_guest_graphical_x86_64.xml

```diff
@@ -126,42 +126,54 @@
     <services>
       <disable config:type="list"/>
       <enable config:type="list">
+        <service>cron</service>
+        <service>display-manager</service>
+        <service>nscd</service>
+        <service>postfix</service>
+        <service>purge-kernels</service>
+        <service>rollback</service>
+        <service>rsyslog</service>
         <service>sshd</service>
+        <service>wicked</service>
+        <service>wickedd-auto4</service>
+        <service>wickedd-dhcp4</service>
+        <service>wickedd-dhcp6</service>
+        <service>wickedd-nanny</service>
+        <service>YaST2-Firstboot</service>
+        <service>YaST2-Second-Stage</service>
+        <service>getty@tty1</service>
       </enable>
     </services>
   </services-manager>
   <software>
+    <install_recommended t="boolean">true</install_recommended>
     <products config:type="list">
         <product>SLES</product>
     </products>
     <packages config:type="list">
+      <package>snapper</package>
       <package>sles-release</package>
-      <package>sle-module-web-scripting-release</package>
-      <package>sle-module-server-applications-release</package>
-      <package>sle-module-legacy-release</package>
-      <package>sle-module-desktop-applications-release</package>
-      <package>sle-module-basesystem-release</package>
-      <package>less</package>
-      <package>wicked</package>
       <package>openssh</package>
-      <package>numactl</package>
       <package>kexec-tools</package>
-      <package>irqbalance</package>
-      <package>iproute2</package>
-      <package>grub2</package>
       <package>glibc</package>
       <package>e2fsprogs</package>
-      <package>chrony</package>
+      <package>dosfstools</package>
       <package>btrfsprogs</package>
-      <package>biosdevname</package>
-      <package>autoyast2</package>
     </packages>
     <patterns config:type="list">
+      <pattern>32bit</pattern>
+      <pattern>Minimal</pattern>
       <pattern>base</pattern>
-      <pattern>basesystem</pattern>
-      <pattern>minimal_base</pattern>
-      <pattern>gnome_basic</pattern>
+      <pattern>documentation</pattern>
+      <pattern>gnome-basic</pattern>
+      <pattern>sles-Minimal-32bit</pattern>
+      <pattern>sles-apparmor-32bit</pattern>
+      <pattern>sles-base-32bit</pattern>
+      <pattern>sles-documentation-32bit</pattern>
+      <pattern>sles-x11-32bit</pattern>
+      <pattern>sles-yast2</pattern>
```

Comment:
why so many 32bit patterns are needed? Our guests are all 64bit, right?

Reply:
Yes, 32bit is installed by default installation as well besided 64bit. And yes, all guests are 64bit. All autoyast settings here come from standard document and facts. So I do not recommend to change them again unless you have very strong reason or actual experience or something happens.

Reply:
Can you share which standard document these 32bit settings come from?

Reply:
Can you share which standard document these 32bit settings come from?

Reply:
This comes from fact. You can install a similar one then verify your guest using "yast2 clone_system".

Reply:
Okay, I see how they come now. 
yast2 clone_system will give all settings based on "current" sles code/packages. It is quite possible that some packages are removed from default install list after some code change, or some services become disabled/enabled. But with current guest yast files(strictly static), they will always require the vm installed in the old way even if default behaviors(pkg/service) change. Then we will be pushed into a situation to adapt the autoyast files frequently to follow code changes. This is not recommended.

An easier way to let us maintain them is to define only NECESSARY parts, and let other parts(that are not explicitly stated in autoyast) following default behavior.

Reply:
Actually we always install 32bit pattern even with autoyast file from qa_lib_virtauto package. They are installed by default onto 12-SP5 guest. And it is very unlikely to change, because these packages are for backward compatibility for you to run 32bit application on 64bit platform. I recommend to keep this way, just as what we got from "yast2 clone_system"

Reply:
My major point in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13146/files#r707838291 is not for 32bit or not, but the way that we configure autoyast -- an easier way to let us maintain them is to define only NECESSARY parts, and let other parts(that are not explicitly stated in autoyast) following default behavior, so not the complete file generated by  yast2 clone_system.

This PR is titled "new and improved provision for xx", I am not sure whether you use this way to generate all the autoyast files and take them as improved version. If yes, sorry, I will need to strongly reject the improvement this time, because the direction leads us to potential time to time maintain work if pkg/svc list change.

Reply:
Firstly, this pr includes "new and improved provisions" which matches what I do in it. And I did not generate new autoyast files. Your compliant is not suitable. Secondly, it is very unlikely that there will be change to these patterns. Patterns are relatively stable compare with packages. Let alone 32bit is definitely supported as per 12-SP5 document. Thirdly, I understand your point. If you insist, I can remove some that are actually installed by default but looks quite unnecessary. Done.

Reply:
Yes, please change. I do not see further commits after the above comment.  Will hold on merge until it is done.

Reply:
It is already changed.